lib/net/packets.js

```diff
@@ -62,6 +63,12 @@ exports.types = {
   GETBLOCKTXN: 24,
   BLOCKTXN: 25,
   UNKNOWN: 26,
+  GETCFILTERS: 29,
+  CFILTER: 30,
+  GETCFHEADERS: 31,
+  CFHEADERS: 32,
+  GETCFCHECKPT: 33,
+  CFCHECKPT: 34,
```

Comment:
sorry what i meant here is something like this...

I don't think the numbers themselves are consensus critical right? it's just an internal enum, so bumping up internal data, unknown, etc shouldn't cause any problems.

```
...
  FEEFILTER: 21,
  SENDCMPCT: 22,
  CMPCTBLOCK: 23,
  GETBLOCKTXN: 24,
  BLOCKTXN: 25,
  GETCFILTERS: 26,
  CFILTER: 27,
  GETCFHEADERS: 28,
  CFHEADERS: 29,
  GETCFCHECKPT: 30,
  CFCHECKPT: 31,
  // Internal
  UNKNOWN: 32,
  INTERNAL: 33,
  DATA: 34
```